principles/business-architecture-principles.md

```diff
@@ -0,0 +1,71 @@
+---
+category: Business Architecture Principles
+expires: 2025-08-31
+---
+# Business Architecture Principles
+
+Business Architecture is a structured approach for aligning an organisation’s strategy with its operational delivery. For DfE it can provide insight into how DfE’s capabilities, its organisation and information can be best structured and utilised to bring value through the delivery of DfE’s public service mission and more broadly of transparency, accessibility, sustainability and the efficient use of resources.
```

Comment:
I think this is the wrong way round. Bus Arch is an approach to aligning and optimising it's operational delivery to the organisation's strategy

Reply:
business architecture often plays a key role in recommending strategic adjustments to align with operational delivery so it can be both ways around

Reply:
I could add "or vice versa"?

Reply:
mind you work might also recommend changes in both so both end up moving.. what about "Business Architecture is a structured approach that aims to achieve alignment between an organisation’s strategy and its operational delivery"

Reply:
I think it's definitely the strategy>delivery way not vice-versa. If delivery isn't working then you may revisit your strategy to deliver differently, but I'd take that as continuous (target) architecture improvement ("closing the gap") rather than aligning strategy with operational delivery. It's not that what's working or not doesn't inform strategic decisions (the evidence), but that it is still a flow from strategy to delivery.

Reply:
> mind you work might also recommend changes in both so both end up moving.. what about "Business Architecture is a structured approach that aims to achieve alignment between an organisation’s strategy and its operational delivery"

this is definitely better